templates/security/certifications.html

```diff
@@ -15,14 +15,14 @@
   <section class="p-strip--suru-topped">
     <div class="row u-equal-height u-vertically-center">
       <div class="col-8">
-        <h1>Security Certifications & Hardening</h1>
+        <h1>Security compliance</h1>
         <h2 class="p-heading--3">Run regulated and high security workloads on Ubuntu</h2>
         <p>
-          Whatever cybersecurity framework you have chosen, including ISO 27000, NIST, PCI or CIS Controls, <a href="/pro">Ubuntu Pro on-premise</a> or on <a href="/public-cloud">public clouds</a> enables your compliance and reduces your operational risk. Access automation for hardening and compliance profiles, such as CIS and DISA-STIG as well as the FIPS 140-2 and Common Criteria certifications.
+          <a href="/pro">Ubuntu Pro</a> has been designed to simplify your security compliance burden for frameworks such as NIST, FedRAMP, PCI-DSS, ISO27001 or CIS. Pro includes security vulnerability patching for up to 12 years, FIPS-validated cryptographic modules, and automated system hardening for CIS and DISA STIG, and can be deployed on-premise or in the <a href="/public-cloud">public cloud</a>.
         </p>
         <p>
           <a class="p-button--positive js-invoke-modal"
-             href="/security/contact-us">Contact us</a>
+             href="/security/certifications#get-in-touch">Contact us</a>
```

Comment:
It would be better here to keep the `/security/contact-us` as the `href` acts as a fallback for when the JS modal fails.